engine/api/worker_model.go

```diff
@@ -76,6 +76,19 @@ func (api *API) addWorkerModelHandler() Handler {
 	}
 }
 
+func (api *API) bookWorkerModelHandler() Handler {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		workerModelID, errr := requestVarInt(r, "permModelID")
+		if errr != nil {
+			return sdk.WrapError(errr, "updateWorkerModel> Invalid permModelID")
+		}
+		if _, err := worker.BookForRegister(api.Cache, workerModelID, getHatchery(ctx)); err != nil {
+			return sdk.WrapError(err, "updateWorkerModel> worker model already booked")
+		}
+		return WriteJSON(w, r, nil, http.StatusOK)
```

Comment:
dont write json if there is nothing to write. 
return nil and i guess the status code will be 204